code/game/dna/genes/goon_powers.dm

```diff
@@ -12,16 +12,16 @@
 	block = SOBERBLOCK
 
 //WAS: /datum/bioEffect/psychic_resist
-/datum/dna/gene/basic/psychic_resist
+/datum/dna/gene/basic/interference
 	name = "Psy-Resist"
```

Comment:
Isn't it weird you didn't change the name here even though you did on the injector?

Reply:
yes